recipe/patches/0002-BLD-Find-nvimgcodec-package-instead-of-downloading-s.patch

```diff
@@ -0,0 +1,37 @@
+From 339a04a3f0d053dba1fc48db61edef237a90f8e1 Mon Sep 17 00:00:00 2001
+From: Daniel Ching <dching@nvidia.com>
+Date: Fri, 18 Apr 2025 18:07:15 -0500
+Subject: [PATCH 2/5] BLD: Find nvimgcodec package instead of downloading
+ source
+
+---
+ cmake/Dependencies.common.cmake | 13 +------------
+ 1 file changed, 1 insertion(+), 12 deletions(-)
+
+diff --git a/cmake/Dependencies.common.cmake b/cmake/Dependencies.common.cmake
+index 9f4a404c..4448bf03 100644
+--- a/cmake/Dependencies.common.cmake
++++ b/cmake/Dependencies.common.cmake
+@@ -293,18 +293,7 @@ if(BUILD_NVIMAGECODEC)
+     endif()
+ 
+     # Note: We are getting the x86_64 tarball, but we are only interested in the headers.
+-    include(FetchContent)
```

Comment:
Can you merge these two codepaths and upstream in DALI - if we fail with `find_package` we can fallback to  `FetchContent`.